bin/node/src/commands.rs

```diff
@@ -75,17 +98,10 @@ fn backup_path(base_path: &Path, backup_dir: &str) -> PathBuf {
 
 fn open_keystore(
     keystore_params: &KeystoreParams,
-    chain_params: &ChainParams,
-    account_id: &AccountId,
+    chain_id: &str,
+    base_path: &Path,
 ) -> impl SyncCryptoStore {
-    let chain_id = chain_params.chain_id();
-    let base_path: BasePath = chain_params
-        .base_path()
-        .path()
-        .join(account_id.to_string())
-        .into();
-
-    let config_dir = base_path.config_dir(chain_id);
+    let config_dir = base_path.join("chains").join(chain_id);
```

Comment:
I re-introduced `BasePath`. We still want to use `PathBuf` for `clap`-defined structs with a subsequent conversion, since we probably do not want to implement traits for `BasePath`.